modules/ROOT/pages/workspaces-and-repositories.adoc

```diff
@@ -298,3 +298,10 @@ To restore an older point:
 . Go to the *Team* > *SVN*, then *Manage restore points*.
 . Select *Restore to an older point* then click *_Next_*.
 . Select the restore point you want to restore, then click *_Restore_*. The project reverts to the restore point.
+
+== image:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
+
+NOTE: Since 2021.2-u1
+
+**My Studio crashed and I've lost my projects in my workspace.** +
```

Comment:
I suggest to use a lvl 3 title here instead of a blod title -> a real title appears in the table of content and shows up quickly with the search. It's what you want for troublshooting
```suggestion
=== My Studio crashed and I've lost my projects in my workspace
```